ansible/playbook/tasks/update.yml

```diff
@@ -20,18 +20,28 @@
       - dbaas-controller
       - dbaas-tools
       - pmm2-client
+    container_types:
+    - docker
+    - podman
+    - lxc
+    - container
+  pre_tasks:
+    - name: detect containers
+      set_fact:
+        is_container: '{{ True if (ansible_virtualization_type in container_types or (ansible_env["container"] is defined and ansible_env["container"] in container_types)) else False }}'
```

Comment:
Can we simplify that? Could it be just `ansible_virtualization_type in container_types or ansible_env.get("container") in container_types`?

Can we also rely on the content of `/srv/pmm-distribution` file? If it is "docker", then that's a container.

Reply:
> Can we simplify that? 
Yes, it can be simplified

> Can we also rely on the content of /srv/pmm-distribution file?
Well, that isn't a question for me :) ... but, considering it is in a persistent volume I'm not sure that it is reliable. That volume could be copied, edited, etc

Reply:
Some other things depend on `/srv/pmm-distribution` being present, so I would say let's do that. It would be less fragile than trying to keep up with new container runners appearing